### ISSUE TITLE
Speedup the UpdateStats window

### ISSUE THREAD

**Issue by ValarDragon** [cometbft/cometbft#2956]
### Summary

Peer gossip does an exponential moving average for choosing what channels to gossip on. Its currently set to decrementing the tracker by 20% every 2 seconds.

Consensus on Osmosis in its entirety takes 1 second, and were at block time 3s, soon to be 2.5s. So I think this EMA is very misparameterized. Instead I suggest making the updateStats tracker happen every 500ms. This is done by changing this constant right here: https://github.com/cometbft/cometbft/blob/main/p2p/conn/connection.go#L34

I think we should observe network behavior to determine if we should adjust this further

**Comment by cason** [comment]
Can you rephrase the title of the issue? And also make it clearer the goal?
